examples/scripts/kto.py

```diff
@@ -15,22 +15,41 @@
 """
 Run the KTO training script with the following command with some example arguments:
 
+# regular:
 python examples/scripts/kto.py \
-    --model_name_or_path "gpt2" \
-    --per_device_train_batch_size 8 \
-    --gradient_accumulation_steps 2 \
-    --learning_rate 1e-4 \
+    --model_name_or_path=gpt2 \
+    --per_device_train_batch_size 4 \
     --max_steps 1000 \
-    --report_to "wandb" \
-    --gradient_checkpointing True  \
-    --output_dir="./test" \
-    --use_peft True \
-    --lora_r 64 \
-    --lora_alpha 16 \
-    --evaluation_strategy "steps" \
-    --logging_first_step True \
+    --learning_rate 1e-3 \
```

Comment:
For my general understanding, are large learning rates like this the typical values for KTO runs? I noticed in your [config](https://github.com/ContextualAI/HALOs/blob/f9f78263cb7aa70809a87f9f74482dfd09f8c60b/config/config.yaml#L55) the you have 5e-7 which is closer to what I expect.

We aim for these simple examples to "just work", so having a good default here would be great

Reply:
I just copied over the suggested commands in scripts/dpo.py and changed dpo -> kto. The command for DPO also seems to use a different learning rate than the 5e-7 that the original DPO uses?

All our experiments in the paper used the same training configs for DPO and KTO, so I'd recommend they be the same here as well (what exactly those configs are gpt2, I don't have a strong opinion on).